scylla/src/errors.rs

```diff
@@ -291,12 +296,15 @@ impl ConnectionError {
 }
 
 /// Error caused by failed address translation done before establishing connection
-#[derive(Debug, Copy, Clone, Error)]
+#[derive(Debug, Clone, Error)]
 pub enum TranslationError {
```

Comment:
I think it may be beneficial in the future to have this marked as `non_exhaustive`. WDYT?

Reply:
Sure. What about `[Query/NewSession]Error`s?

Reply:
Them both definitely.